app/src/main/java/org/p2p/wallet/auth/model/CountryCode.kt

```diff
@@ -1,16 +1,24 @@
 package org.p2p.wallet.auth.model
 
 import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
 import kotlinx.parcelize.Parcelize
+import org.p2p.core.utils.emptyString
 
 @Parcelize
 data class CountryCode(
+    @SerializedName("code_alpha_2")
```

Comment:
if you want to keep this data in Shared Prefs - create a separate entity for json please